Editor/UI/AnalysisView.cs

```diff
@@ -192,23 +192,30 @@ public void OnGUI()
         void DrawTable(ProjectIssue[] selectedIssues)
         {
             EditorGUILayout.BeginVertical();
+            EditorGUILayout.Space();
 
-            var r = EditorGUILayout.GetControlRect(GUILayout.ExpandHeight(true));
-
-            Profiler.BeginSample("IssueTable.OnGUI");
-            m_Table.OnGUI(r);
-            Profiler.EndSample();
-
+            EditorGUILayout.BeginHorizontal();
             if (m_Desc.groupByDescription)
             {
-                EditorGUILayout.BeginHorizontal();
-                if (GUILayout.Button(Styles.CollapseAllButton, GUILayout.ExpandWidth(true), GUILayout.Width(100)))
+                if (GUILayout.Button(Styles.CollapseAllButton, EditorStyles.toolbarButton, GUILayout.ExpandWidth(true), GUILayout.Width(100)))
                     SetRowsExpanded(false);
-                if (GUILayout.Button(Styles.ExpandAllButton, GUILayout.ExpandWidth(true), GUILayout.Width(100)))
+                if (GUILayout.Button(Styles.ExpandAllButton, EditorStyles.toolbarButton, GUILayout.ExpandWidth(true), GUILayout.Width(100)))
```

Comment:
If the intent is to keep this PR easily revertible when/if Unity supports font size changes, adding the `EditorStyles.toolbarButton` param should go in another PR.

Reply:
You are absolutely right. However, in this case, I am not going to bother simply because there are more changes coming that will affect this code. Removing this feature will be simple enough anyway. Thanks!